core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/SequencerPartitionContainerImpl.java

```diff
@@ -405,4 +371,14 @@ public ChangeLog createChangeLog(String summary, String columnsChanged, User use
     changeLog.setUser(user);
     return changeLog;
   }
+
+  @Transient
+  private final Comparator<Partition> partitionNumberComparator = new Comparator<Partition>() {
```

Comment:
It's funny that I went "why is Hibernate trying to save this?" and didn't think of that